.gitignore

```diff
@@ -2,6 +2,8 @@
 /release
 /sht3x/sht3x_example_usage_hw_i2c
 /sht3x/sht3x_example_usage_sw_i2c
+/sht3x/user_config.inc
```

Comment:
I think the suffix `_hw_i2c` and `_sw_i2c` is gone

Reply:
Good catch!